.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+   - "0.12.0"
+
+branches:
```

Comment:
You should remove this until the tests are finished. I'm not sure you'll want to test only dev and master.